packages/app/universal-ts-utils/src/array/areArraysEqual.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Check if two arrays are equal.
+ * The arrays are considered equal if they have the same length, same order and the elements at each index by value are equal.
+ * Note: Equality is compared by value and not by reference.
+ */
+export function areArraysEqual(a: unknown[], b: unknown[]): boolean {
```

Comment:
Thinking about having `areEquals` so it can be also used to compare objects, still thinking about it

Reply:
from perf perspective separate functions are better, because isomorphic params (different structure across multiple calls) will cause v8 to deoptimize.

Reply:
I am not sure, my idea was to have something like:
```
areEquals(a: unknown, b: unknown): boolean
```
it won't use isomorphic params but not sure if v8 will deoptimize it

Reply:
As an example -> https://github.com/lokalise/shared-ts-libs/pull/331/commits/db19fb850f9dcd797289e38e4911f47557dad09a